.github/workflows/brakeman.yml

```
@@ -20,12 +20,34 @@ jobs:
         with:
           bundler-cache: true
 
-      - name: Run Brakeman
+      - name: Run Brakeman SARIF
         continue-on-error: true
+        if: ${{ github.repository_visibility == 'public' }}
         run: bundle exec brakeman . --except CheckRenderInline --quiet -f sarif >> brakeman.sarif
 
-      - name: Upload result to Github Code Scanning
-        if: always()
+      - name: Run Brakeman JSON
+        continue-on-error: true
+        if: ${{ github.repository_visibility == 'private' }}
```

We also want any internal repos to output JSON. For example govuk-rota-announcer 
```suggestion
        if: ${{ github.repository_visibility == 'private'  || github.repository_visibility == 'internal' }}
```